### PR TITLE
Add EtherInc node

### PR DESCRIPTION
**Pull request by suryanshkumr**
# Description
EtherInc is a fork of Ethereum to power the future of blockchain based organizations.

* Homepage: https://einc.io/
* Node: https://api.einc.io/jsonrpc/mainnet
* Explorer: https://explorer.einc.io/
* BIP44 / Chainid 101 ( satoshilabs/slips#438 )

# Changes
This PR adds EtherInc node support

### CODE REVIEW

File: common/features/config/networks/static/reducer.ts [hunk 1]
```diff
@@ -698,6 +699,58 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
       max: 60,
       initial: 20
     }
+  },
+
+  ETI: {
+    id: 'ETI',
+    name: 'ETI',
+    unit: 'ETI',
+    chainId: 101,
```
Comment (mkrufky) [inline common/features/config/networks/static/reducer.ts]: @suryanshkumr  Does ETI support EIP-155?

Regardless of whether EIP-155 is actually supported or not, chainId 101 is already in use by WebChain Network.

It is not safe for both networks to use the same chainId - I'd suggest a hardfork ASAP to pick a chainId that is unique to EtherInc

File: common/features/config/networks/static/reducer.ts [hunk 2]
```diff
@@ -698,6 +699,58 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
...
+    isCustom: false,
+    color: '#3560bf',
+    blockExplorer: makeExplorer({
+      name: 'EtherInc Explorer',
+      origin: 'https://explorer.einc.io'
+    }),
+    tokens: [],
+    contracts: [],
+    dPathFormats: {
+      [SecureWalletName.TREZOR]: ETI_DEFAULT,
```
Comment (mkrufky) [inline common/features/config/networks/static/reducer.ts]: Please link to your Trezor PR, or remove the Trezor support if it isn't actually supported.

File: common/features/config/networks/static/reducer.ts [hunk 3]
```diff
@@ -698,6 +699,58 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
...
+      [SecureWalletName.SAFE_T]: ETI_DEFAULT,
+      [SecureWalletName.LEDGER_NANO_S]: ETI_DEFAULT,
```
Comment (mkrufky) [inline common/features/config/networks/static/reducer.ts]: Please link to your Ledger PR, or remove the Ledger support if it isn't actually supported.

File: common/features/config/networks/static/reducer.ts [hunk 4]
```diff
@@ -698,6 +699,58 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
...
+    color: '#3560bf',
+    blockExplorer: makeExplorer({
+      name: 'EtherInc Ropsten Explorer',
+      origin: 'https://ropstenexplorer.einc.io'
+    }),
+    tokens: [],
+    contracts: [],
+    isTestnet: true,
+    dPathFormats: {
+      [SecureWalletName.TREZOR]: ETI_DEFAULT,
```
Comment (mkrufky) [inline common/features/config/networks/static/reducer.ts]: Please link to your Trezor PR, or remove the Trezor support if it isn't actually supported.

File: common/features/config/networks/static/reducer.ts [hunk 5]
```diff
@@ -698,6 +699,58 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
...
+      [SecureWalletName.SAFE_T]: ETI_DEFAULT,
+      [InsecureWalletName.MNEMONIC_PHRASE]: ETI_DEFAULT
```
Comment (mkrufky) [inline common/features/config/networks/static/reducer.ts]: No Ledger Nano S support on TestNet?  Do you want your users to use Ledger in production, but they can't test with it?  What if something goes wrong?

**Comment by tayvano** [comment]
See response here: https://github.com/MyCryptoHQ/MyCrypto/pull/2217

**Comment by suryanshkumr** [comment]
@tayvano WebChain did a fork and changing its chain ID

**Comment by tayvano** [comment]
Still seeing Webchain as using chainid 101 here: https://github.com/MyEtherWallet/MyEtherWallet/blob/develop/src/networks/types/WEB.js

Do you have a source for their update?

**Comment by suryanshkumr** [comment]
@tayvano https://github.com/webchain-network/webchaind/pull/2

**Pull request closed without merging**